data/hai missions.txt

```diff
@@ -473,3 +473,428 @@ mission "Unfettered returning home"
+	source "Vara K'chrai"
+	destination Greenwater
+	cargo "Eruks equipment" 21
+	passengers 1
+	autosave
+	to offer
+		has "event: eruk collects wanderer data" 
+	on offer
+		"reputation: Unidentified" = -1000
+		conversation
+			`Coming out of your ship you notice that the Redacted is not a few pads over like the last few times you have landed. You shove it to the back of your mind as you head over to the Hai embassy, where Eruk has been staying over the past year.`
+			`	Outside the embassy there is a pair of that Wanderer trucks being loaded up, though you can't see any sign of Eruk. At the front door you stand aside to let a pair of Wanderers carrying what appears to be a large crate of data drives stride out followed by Eruk carrying a large clipboard.`
+			`	"Ah, <first>, I assume you are prepared to transport me back to Greenwater."`
+			choice
+				`	"yes, of course."`
+					goto yes
+				`	"Sorry, I don't think I can at the moment."`
+			`	"Don't worry about it <first>, the Wanderers have already said they would lend me one of their ships with a Jump Drive to get me back to Hai space."`
+			`	Back at your ship you notice that Eruk's equipment being loaded onto one of the Wanderers Strong Wind warships.`
+				decline
+
+			label yes
+			`	"Ecellent, excellent. I'll tell them to start loading up immediately."`
+				accept
+
+	npc evade
+		personality nemesis unconstrained entering
+		government Unidentified
+		ship "Korath Raider (Overclocked)" "Redacted"
+
+	npc evade
+		personality nemesis unconstrained
+		government Unidentified
+		system "Pug Iyik"
+		ship "Korath Raider (Overclocked)" "Redacted"
+
+	npc evade
+		personality nemesis unconstrained
+		government Unidentified
+		system "Aya'k'k"
+		ship "Korath Raider (Overclocked)" "Redacted"
+
+	npc evade
+		personality nemesis unconstrained
+		government Unidentified
+		system "Prakacha'a"
+		ship "Korath Raider (Overclocked)" "Redacted"
+
+	npc evade
+		personality nemesis unconstrained
+		government Unidentified
+		system "Mintaka"
+		ship "Korath Raider (Overclocked)" "Redacted"
+
+	npc evade
+		personality nemesis unconstrained
+		government Unidentified
+		system "Lom Tahr"
+		ship "Korath Raider (Overclocked)" "Redacted"
+
+	on visit "Some of the unidentified ships are still in orbit. You will have to take off again to try and lose them."
+
+	on complete
+		payment 1000000
+		conversation
+			`Eruk seems to be a bit rattled from the attacks you had on your way here. "I can't think who would want me dead for the work I've been doing. Hopefully they won't be able to find me, now you've got me safely here." He looks very worried. "I should have this software sorted in a couple of months, so when I do I'll send a message telling you how to update. Also, here's a bit of a bonus for keeping me alive." he hands you a 1 million credit chip. "If I ever need you again, I'll give you a call." and with that, he walks off.'`
```

Comment:
Hopefully, they won't (or: "now that you've got me safely here")

He needs to be capitalized.

So when I do, I'll